site/viamguides/src/pi-ai-hat/pi-ai-hat.md

```diff
@@ -0,0 +1,305 @@
+author: HipsterBrown
+id: pi-ai-hat
+summary: Use real-time object detection with the Raspberry Pi AI HAT
+categories: Getting-Started, Developer
+environments: web
+status: Published 
+feedback link: https://github.com/viam-labs/viamcodelabs/issues
+tags: Getting Started, Developer
+
+# Real-Time Object Detection with Raspberry Pi AI HAT
+<!-- ------------------------ -->
+## Overview 
+Duration: 1
```

Comment:
Should we add the duration units (minutes, hours)? This goes for all the duration attributes.

Reply:
While I agree on the added clarity, the `claat` (codelabs "compiler") CLI can't parse the duration field if any units are added. The implicit unit is minutes.

Reply:
I see, thanks for clarifying!